include/dma.h

```diff
@@ -34,6 +34,13 @@ enum dma_channel_direction {
 	PERIPHERAL_TO_MEMORY
 };
 
+/** Valid values for @a source_addr_adj and @a dest_addr_adj */
```

Comment:
This should be its own commit, and its own PR since you are 'updating' the API.

Reply:
Split out.

Reply:
annnd now actually split out into #6615